### PR TITLE
capi: Export inspection functions

### PR DESCRIPTION
**Pull request by gumb0**

### CODE REVIEW

File: include/fizzy/fizzy.h [hunk 1]
```diff
@@ -133,6 +133,27 @@ typedef struct FizzyExternalGlobal
...
+typedef enum FizzyExternalKind
+{
+    FizzyExternalKindFunction,
+    FizzyExternalKindTable,
+    FizzyExternalKindMemory,
+    FizzyExternalKindGlobal
+} FizzyExternalKind;
+
+/// Export description.
+typedef struct FizzyExportDescription
```
Comment (axic) [inline include/fizzy/fizzy.h:146]: Same comment as with FizzyImportDescription: https://github.com/wasmx/fizzy/pull/683#discussion_r564742640. Please add a remark about the lifetime of `name`.

File: include/fizzy/fizzy.h [hunk 2]
```diff
@@ -133,6 +133,27 @@ typedef struct FizzyExternalGlobal
     FizzyGlobalType type;
 } FizzyExternalGlobal;
 
+/// External kind.
+typedef enum FizzyExternalKind
```
Comment (axic) [inline include/fizzy/fizzy.h:137]: Oh, this is in conflict with #683, so need to one first then rebase.

File: include/fizzy/fizzy.h [hunk 3]
```diff
@@ -222,6 +243,23 @@ uint32_t fizzy_get_global_count(const FizzyModule* module);
 /// @note  All module global indices are greater than all imported global indices.
 FizzyGlobalType fizzy_get_global_type(const FizzyModule* module, uint32_t global_idx);
 
+/// Get number of exports defined in the module.
+///
+/// @param  module    Pointer to module. Cannot be NULL.
+/// @return           Number of exports in the module.
+uint32_t fizzy_get_export_count(const FizzyModule* module);
+
+/// Get the import description defined in the module.
```
Comment (axic) [inline include/fizzy/fizzy.h:252]: Import?

**Review by axic — APPROVED** [review]
Looks good too, just some of those documentation questions.

File: include/fizzy/fizzy.h [hunk 4]
```diff
@@ -133,6 +133,27 @@ typedef struct FizzyExternalGlobal
...
+    FizzyExternalKindFunction,
+    FizzyExternalKindTable,
+    FizzyExternalKindMemory,
+    FizzyExternalKindGlobal
+} FizzyExternalKind;
+
+/// Export description.
+typedef struct FizzyExportDescription
+{
+    /// Export name
```
Comment (gumb0) [inline include/fizzy/fizzy.h:148]:
```suggestion
    /// Export name.
```

File: include/fizzy/fizzy.h (hunk 3 above)
Comment (gumb0) [reply include/fizzy/fizzy.h]: Fixed.

File: include/fizzy/fizzy.h (hunk 1 above)
Comment (gumb0) [reply include/fizzy/fizzy.h:179]: Added.

File: include/fizzy/fizzy.h [hunk 5]
```diff
@@ -172,6 +172,21 @@ typedef struct FizzyImportDescription
...
+/// @note For the lifetime of the #name field, please refer to the description provided by the
+/// function used to obtain this structure.
+typedef struct FizzyExportDescription
+{
+    /// Export name.
+    const char* name;
+    /// Export kind.
+    FizzyExternalKind kind;
+    /// Index of exported function or table or memory or global.
+    /// #kind determines what is this index of.
```
Comment (axic) [inline include/fizzy/fizzy.h:186]:
```suggestion
    /// #kind determines what is this the index of.
```

**Review by axic — APPROVED** [review]

**Pull request merged**